src/library/scala/collection/BitSetLike.scala

```diff
@@ -204,6 +204,27 @@ trait BitSetLike[+This <: BitSetLike[This] with SortedSet[Int]] extends SortedSe
   def subsetOf(other: BitSet): Boolean =
     (0 until nwords) forall (idx => (this.word(idx) & ~ other.word(idx)) == 0L)
 
+  override def head: Int = {
```

Comment:
The implementation of this algorithm is slightly different from what `BitSetLike.foreach` does. Should they be the same? Which one is faster?